mu4/notation/internal/notationactioncontroller.cpp

```diff
@@ -252,6 +325,25 @@ void NotationActionController::swapVoices(int voiceIndex1, int voiceIndex2)
     interaction->swapVoices(voiceIndex1, voiceIndex2);
 }
 
+void NotationActionController::changeVoice(int voiceIndex)
+{
+    auto interaction = currentNotationInteraction();
+    if (!interaction) {
+        return;
+    }
+
+    auto noteInput = interaction->noteInput();
+    if (!noteInput) {
+        return;
+    }
+
+    if (noteInput->isNoteInputMode()) {
+        noteInput->setCurrentVoiceIndex(voiceIndex);
```

Comment:
It seems that this should always be done.

Reply:
fixed